CONTRIBUTORS.md

```diff
@@ -2,22 +2,29 @@
 
 We contributors to System Initiative:
 
-* License all our contributions to the project under the Apache License, Version 2.0
-* Have the legal rights to license our contributions ourselves, or get permission to license them from our employers, clients, or others who may have them
-* Add our names and GitHub handles to this CONTRIBUTORS.md file to create a permanent record that users, distributors, and other contributors can all rely on
+- License all our contributions to the project under the Apache License, Version
+  2.0
+- Have the legal rights to license our contributions ourselves, or get
+  permission to license them from our employers, clients, or others who may have
+  them
+- Add our names and GitHub handles to this CONTRIBUTORS.md file to create a
+  permanent record that users, distributors, and other contributors can all rely
+  on
 
------------
-* Adam Jacob (@adamhjk)
-* Anna Saulwick (@AnnaAtMax)
-* Brit Myers (@britmyerss)
-* Fletcher Nichol (@fnichol)
-* Jacob Helwig (@jhelwig)
-* Mahir Lupinacci (@mahirl)
-* Nick Gerace (@nickgerace)
-* Paul Stack (@stack72)
-* Paulo Cabral (@paulocsanz)
-* Theo Ephraim (@theoephraim)
-* Victor Bustamante (@vbustamante)
-* Wendy Bujalski (@wendybujalski)
-* Zack Hamm (@zacharyhamm)
-* Dan Miller (@jazzdan)
+---
+
+- Adam Jacob (@adamhjk)
+- Anna Saulwick (@AnnaAtMax)
+- Brit Myers (@britmyerss)
+- Fletcher Nichol (@fnichol)
+- Jacob Helwig (@jhelwig)
+- Mahir Lupinacci (@mahirl)
+- Nick Gerace (@nickgerace)
+- Paul Stack (@stack72)
+- Paulo Cabral (@paulocsanz)
+- Theo Ephraim (@theoephraim)
+- Victor Bustamante (@vbustamante)
+- Wendy Bujalski (@wendybujalski)
+- Zack Hamm (@zacharyhamm)
+- Dan Miller (@jazzdan)
+- Neil Hanlon (@NeilHanlon)
```

Comment:
Did changing the markdown from `*` to `-` break the contributors check here?

Reply:
oops - sorry about that. will check my linter/formatter